### ISSUE TITLE
p-steps does not respect the disabled attribute on MenuItem

### ISSUE THREAD

**Issue by mlofjard** [primefaces/primeng#2607]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```
**Current behavior**
If `readonly` is set to `false` on p-steps. setting `disabled: true` on a MenuItem used by StepsModule does not disable the step.

**Expected behavior**
Step should not be clickable/enabled. 

**What is the motivation / use case for changing the behavior?**
I would like to be able to enable/disable steps by toggling the `disabled` boolean on a MenuItem.

* **Angular version:** 4.0.1

* **PrimeNG version:** 4.0.0-rc2

**Comment by rachelbr** [comment]
I am so interested too.

**Comment by sudheerj** [comment]
The disabled functionality is used through readonly property in steps component.Better it can be changed to disabled attribute or provide control to the menu item instead.

**Comment by sudheerj** [comment]
I created a PR on this #2718

**Comment by mlofjard** [comment]
Closed as of 4.0.1
